### ISSUE TITLE
Remove `--cert` option from `cadc-get-cert`

### ISSUE THREAD

**Issue by andamian** [opencadc/cadctools#164]
This reveals a weakness in our system. Better to remove it from here now.

**Comment by chitrangpatel** [comment]
Are there plans to replace this system with token or ssh based authentication? That would make life easy for building automated systems to do file uploads.

**Comment by andamian** [comment]
Yes. Support for tokens is being deployed to our services. It will be added to the Python clients after that.
